.github/workflows/lint.yml

```diff
@@ -5,6 +5,7 @@ on:
     paths:
       - src/**
       - .clang-format
+      - .prettierc.json
```

Comment:
This list feels incomplete when looking at this PR's diff

Reply:
In the end I decided to remove the path constraints, and allow `tools/cross/format.py` to format JS files in the entire codebase. If we're worried about that, I could go the opposite way and scope everything to `src/`

Reply:
After all, I'd rather not do excludes like node_modules, etc. so I'm going to switch it to src/

Reply:
Just make sure this list fits whatever is being formatted :)